app/src/main/java/org/vss/impl/postgres/PostgresBackendImpl.java

```diff
@@ -138,10 +138,11 @@ public ListKeyVersionsResponse listKeyVersions(ListKeyVersionsRequest request) {
     // all current key_versions were stored at global_version or later.
     Long globalVersion = null;
     if (!request.hasPageToken()) {
-      globalVersion = get(GetObjectRequest.newBuilder()
+      GetObjectRequest getGlobalVersionRequest = GetObjectRequest.newBuilder()
           .setStoreId(storeId)
           .setKey(GLOBAL_VERSION_KEY)
-          .build()).getValue().getVersion();
+          .build();
+      globalVersion = get(getGlobalVersionRequest).getValue().getVersion();
```

Comment:
Changes here a below belong on the first commit.

Reply:
fixed it 👍